bmarlow/Week 3/mailroom.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+#build our initial dictionary
+dictionary = {
+	'Ricky': '1000',
+	'Bobby': '1100,2200',
+	'Hansel': '5000,6000,7000',
+	'Mr. Burgandy': '1,2,3,4',
+	'Austin': '123,4567,78,99,100',
+	'Dale': '14,234,878,345,345,234'
+}
```

Comment:
Oh god, my powershell is showing.  I didn't even know you could have a dictionary where the value was a list.  That's awesome.

Also, yes dictionary is a terrible name.